src/data_access/FirebaseMessageDataAccessObject.java

```diff
@@ -21,7 +21,11 @@ public void sendMessage(
       Room room, LoginUserDataAccessInterface userDAO, User user, String messageBody) {
     String idToken = userDAO.getAccessToken(user.getEmail(), user.getPassword());
     String messageJSON =
-        new JSONObject().put("contents", messageBody).put("author", user.getEmail()).toString();
+        new JSONObject()
+            .put("contents", messageBody)
+            .put("displayName", user.getEmail())
```

Comment:
why do we need this change?

Reply:
To display the usernames of users in the rooms chat